### PR TITLE
Skip sleep only when vcr_record_mode=none

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/191/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1027/ : SUCCESS in 2m 12s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1296/ : FAILURE in 52s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/192/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1028/ : SUCCESS in 2m 09s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1297/ : SUCCESS in 2m 15s

File: spec/vcr_configurer.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@ module GoodData
   module Helpers
     # Configures VCR for integration tests
     class VcrConfigurer
+      def self.vcr_record_mode
+        (ENV['VCR_RECORD_MODE'] && ENV['VCR_RECORD_MODE'].to_sym) || :none
```
Comment (yenkins) [inline spec/vcr_configurer.rb]: Calls 'env['vcr_record_mode']' 2 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/196/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1031/ : SUCCESS in 51s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1300/ : SUCCESS in 1m 01s

**Comment by panjan** [comment]
tests passed: https://travis-ci.org/panjan/gooddata-ruby/builds/387691793

**Pull request merged**